services/ymax-planner/src/engine.ts

```diff
@@ -248,33 +254,36 @@ const processPortfolioEvents = async (
         return;
       }
       errorContext.steps = steps;
-  
-      const resolvePlanArgs: ResolvePlanArgs = {
-        portfolioId: portfolioIdFromKey(portfolioKey as any),
-        flowId: flowIdFromKey(flowKey as any),
+
+      const portfolioId = portfolioIdFromKey(portfolioKey as any);
+      const flowId = flowIdFromKey(flowKey as any);
+      const { policyVersion, rebalanceCount } = portfolioStatus;
+      const planner = walletStore.get<PortfolioPlanner>('planner', {
+        immediate: true,
```

Comment:
I think of this mode as `sendOnly`. I didn't catch the client-utils PR review in time.

Is "immediate" a more widely used term for it?

Reply:
I don't think it's more widely used, but it does highlight a different aspect. Here's a description of the behavior:
* without `immediate: true`: if `makeNonce` was provided, then an `id` is created and fulfillment of the promise returned from a method invocation waits for confirmation of that `id` in vstorage (after one or more new blocks have been produced), but otherwise it waits only for fulfillment from `sendBridgeAction` (i.e., for tx submission)
* with `immediate: true`: whether or not `makeNonce` was provided, fulfillment of the promise returned from a method invocation waits only for tx submission from `sendBridgeAction`

Given that, it seems like `sendOnly` might fit. Shall I make the change?

Reply:
Done.